doc/security_policies.md

```diff
@@ -0,0 +1,87 @@
+# Security Policies
+
+https://www.suse.com/c/applying-disa-stig-hardening-to-sles-installations/
+
+The DISA ([Defense Information Systems Agency](https://disa.mil)) and SUSE have authored a STIG
+(Secure Technical Implementation Guide) that describes how to harden a SUSE Linux Enterprise system.
+
+The STIG is a long list of rules, each containing description, detection of problems and how to
+remediate problems on a per rule basis.
+
+While originally STIGs are supposed to applied manually, a large percentage of the rules can be and
+were automated in so called SCAP format (Secure Content Automation Protocol).
+
+We can classify rules into multiple cases:
+
+* rules that need to be applied during installation of a system
+* rules where remediation can be automatically applied after installation
+* rules that are not able to be checked automatically nor remediated
+* rules without automated remediation
+
+## YaST and Security Policies
+
+The YaST installer supports security policies which can be enabled in the Installation Summary. The
+installer checks the rules of the enabled policies and reports the issues to fix before proceeding
+with the installation. For some issues, YaST offers a link to easily remediate it. For others, the
+link will take you to the proper installation client to fix the issue (e.g., the Guided
+Partitoning). At this moment, only the DISA STIG policy is supported, but the YaST infrastructure
+is ready to offer more policies if requested.
+
+The boot parameter `YAST_SECURITY_POLICIES` can be used for enabling policies from the beginning of
+the installation without waiting until reaching the Installation Summary. If the policies are
+enabled (e.g., `YAST_SECURITY_POLICIES=disa_stig`), then some YaST clients could show policies
+issues too. For example, the Guided Partitioning client and the Expert Partitioner show the policies
+issues related to storage configuration.
+
+## DISA STIG Checks
+
+The YaST installer only checks a subset of the rules defined by a security policy. YaST is mainly
+focused on such rules that need to be applied during the installation. For example, for DISA STIG
+the following rules should be checked:
+
+* All file systems are encrypted ([SLES-15-010330](http://static.open-scap.org/ssg-guides/ssg-sle15-guide-stig.html#xccdf_org.ssgproject.content_rule_encrypt_partitions])).
```

Comment:
Yay for the identifiers of the rules!

So, the code that checks the rules should reference them, right? Either in a comment or perhaps even in an attribute so that you can run a test and get a list of rules that we implement?

On the contrary, the current implementation in disa_stig_policy.rb is well structured but without referencing the rules which are an actual concept in the STIG.